public/app/features/alerting/unified/components/MenuItemPauseRule.tsx

```diff
@@ -25,22 +24,15 @@ const MenuItemPauseRule = ({ rule, groupIdentifier, onPauseChange }: Props) => {
   const notifyApp = useAppNotification();
   const [pauseRule, updateState] = usePauseRuleInGroup();
 
-  const isPaused = isGrafanaRulerRule(rule) && isGrafanaRulerRulePaused(rule);
```

Comment:
How come we aren't using the helper method btw?

Reply:
The component takes `RulerGrafanaRuleDTO` object instead of `RulerRuleDTO` because pausing is only supported by Grafana rules. Then we can just read the property directly from the rule